mindsdb/make_migrations.py

```diff
@@ -0,0 +1,12 @@
+import os
+from flask import Flask
+from flask_migrate import Migrate
+
+import mindsdb.mindsdb.interfaces.storage.db as db
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = os.environ['MINDSDB_DB_CON']
```

Comment:
if you run that script manually, then how you can be sure MINDSDB_DB_CON is set up? this var could be setup from config file

Reply:
because mindsdb/__init__.py is executed anyway

Reply:
when you run `flask db xxx` you will not have config in args, so it will not be read in _initi__.py

Reply:
What is your suggestion here?

Reply:
Well In case of development it is possible to setup `os.environ['MINDSDB_DB_CON']` manually. 
In case of customer launch with running migration as a part of mindsdb launch, everything will be ok. So I don't see here any serious issues